### ISSUE TITLE
Alcatraz install.sh error  print successfully installed but not

### ISSUE THREAD

**Issue by banxi1988** [alcatraz/Alcatraz#306]
I was using  OS X 10.10.4 (zsh)

```
➜  ~ curl -fsSL https://raw.github.com/supermarin/Alcatraz/master/Scripts/install.sh | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   391    0   391    0     0     51      0 --:--:--  0:00:07 --:--:--    96
  0     0    0   333    0     0      3      0 --:--:--  0:01:26 --:--:--    83
tar: Unrecognized archive format
tar: Error exit delayed from previous errors.
Alcatraz successfully installed!!1!🍻   Please restart your Xcode.
```

I finally  change sh to bash，it's done
* but When I restart My Xcode ,I  cannot find the Package Manager Menu 
* after execute `sudo xcode-select --reset` It works.

```
➜  ~ curl -fsSL https://raw.github.com/supermarin/Alcatraz/master/Scripts/install.sh | sh
^C
➜  ~ curl -fsSL https://raw.github.com/supermarin/Alcatraz/master/Scripts/install.sh | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   393    0   393    0     0    200      0 --:--:--  0:00:01 --:--:--   200
 12  135k   12 16836    0     0   3454      0  0:00:40  0:00:04  0:00:36  8188x Alcatraz.xcplugin/
x Alcatraz.xcplugin/Contents/
x Alcatraz.xcplugin/Contents/Info.plist
x Alcatraz.xcplugin/Contents/MacOS/
x Alcatraz.xcplugin/Contents/Resources/
x Alcatraz.xcplugin/Contents/Resources/740-gear-selected.tiff
x Alcatraz.xcplugin/Contents/Resources/740-gear.tiff
x Alcatraz.xcplugin/Contents/Resources/808-documents-selected.tiff
x Alcatraz.xcplugin/Contents/Resources/808-documents.tiff
x Alcatraz.xcplugin/Contents/Resources/837-palette-selected.tiff
x Alcatraz.xcplugin/Contents/Resources/837-palette.tiff
x Alcatraz.xcplugin/Contents/Resources/ATZPackageListTableCellView.nib
x Alcatraz.xcplugin/Contents/Resources/ATZPluginWindowController.nib
x Alcatraz.xcplugin/Contents/Resources/bitbucket_grayscale.tiff
x Alcatraz.xcplugin/Contents/Resources/en.lproj/
x Alcatraz.xcplugin/Contents/Resources/eye_icon.tiff
x Alcatraz.xcplugin/Contents/Resources/git_grayscale.tiff
x Alcatraz.xcplugin/Contents/Resources/github_grayscale.tiff
x Alcatraz.xcplugin/Contents/Resources/link_icon.tiff
x Alcatraz.xcplugin/Contents/Resources/PluginWindow.nib
x Alcatraz.xcplugin/Contents/Resources/toolbar_colors.tiff
x Alcatraz.xcplugin/Contents/Resources/toolbar_plugins.tiff
x Alcatraz.xcplugin/Contents/Resources/toolbar_templates.tiff
x Alcatraz.xcplugin/Contents/Resources/en.lproj/InfoPlist.strings
x Alcatraz.xcplugin/Contents/Resources/en.lproj/Localizable.strings
100  135k  100  135k    0     0  24881      0  0:00:05  0:00:05 --:--:-- 50131

Alcatraz successfully installed!!1!🍻   Please restart your Xcode.
➜  ~
```

**Comment by 80m** [comment]
![screen shot 2015-07-01 at 12 26 00 pm](https://cloud.githubusercontent.com/assets/45826/8463007/628f4904-1fec-11e5-875c-ea3a170aab1d.png)

**Comment by JamieREvans** [comment]
Is this an OS X `10.10.4` or Xcode `6.4`? I think the title needs to be "Cannot install Alcatraz with Xcode `6.4` and OS X `10.10.4`".

**Comment by JamieREvans** [comment]
Also, I would like to point out it still installs with Xcode Beta, so I believe the issue belongs to Xcode `6.4`.

**Comment by adin283** [comment]
yes,i met the same trouble.Cannot install Alcatraz with Xcode 6.4 and OS X 10.10.4

**Comment by adin283** [comment]
```
// execute first
sudo xcode-select --reset
// and then
curl -fsSL https://raw.github.com/supermarin/Alcatraz/master/Scripts/install.sh | sh
// restart xcode select "load bundle", Alcatraz works
```

**Comment by JamieREvans** [comment]
@adin283 Unfortunately, that doesn't work for me. I think it may have to do with having the Xcode Beta also installed.

**Comment by adin283** [comment]
@JamieREvans 
I haven't install the Xcode Beta,only xcode6.4 in osx10.10.4.

**Comment by danydev** [comment]
It does not work for me either (Xcode 6.4, OS X 10.0.4) I had Xcode 7 beta (don't know if it helps)

**Comment by JamieREvans** [comment]
I've created a new and more specific issue here: https://github.com/supermarin/Alcatraz/issues/316

**Comment by wh1pch81n** [comment]
I have tried @banxi1988  and @adin283 's suggestion.  I can not get the Package Manager Menu to appear.  I am running Xcode 6.4 and the problem sprung up when I upgraded from Xcode 6.3 to 6.4.

**Comment by guillaume-algis** [comment]
@wh1pch81n could you try the following command and report back ?

    rm -rf ~/Library/Application\ Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin/
    curl -fsSL https://raw.githubusercontent.com/guillaume-algis/Alcatraz/feature/whitelist-on-install/Scripts/install.sh | sh

_disclaimer: this uses a custom installer from my fork, not the official one. If you don't trust me (and you shouldn't) I encourage you to read the script before piping it into `sh`_

**Comment by wh1pch81n** [comment]
If I download the github repo by pressing the "download zip" button.  Then open up the Alcatraz project then run it.  Then Xcode tells me that this plug-in is not form apple and gives me the opion to skip or load the bundle.  I click load bundle. The The Package Manager Menu appears.    But after I quit Xcode, and relaunch it, the Package Manager  Menu disappears.

@guillaume-algis I'll look at the script when I get a chance.

**Comment by donevalu** [comment]
What worked for me in Xcode 6.4 - something to do with whitelisting plugins - especially if you installed Xcode6.4 through an upgrade path.

See this link for more details:
http://buegling.com/blog/2015/5/2/xcode-64-plugins-and-you

``` defaults delete com.apple.dt.Xcode DVTPlugInManagerNonApplePlugIns-Xcode-6.4 ```

Then proceed to install Alcatraz again.

**Comment by 0x686169** [comment]
the same on Xcode 7.0 final release

**Comment by l-zhi** [comment]
the same on Xcode 7.2 Beta

**Comment by guillaume-algis** [comment]
Most install script issues referenced here should have been fixed in #315.
